spec/bolt_server/transport_app_spec.rb

```diff
@@ -38,288 +38,259 @@ def file_data(file)
+
+        expect_any_instance_of(BoltServer::TransportApp)
+          .to receive(:scrub_stack_trace).with(result.status_hash).and_return({})
+
+        post(path, JSON.generate(body), 'CONTENT_TYPE' => 'text/json')
 
-        post path, JSON.generate(body), 'CONTENT_TYPE' => 'text/json'
         expect(last_response).to be_ok
         expect(last_response.status).to eq(200)
-        result = JSON.parse(last_response.body)
-        expect(result).to include('status' => 'failure')
-        expect(result['result']['_error']).to include('msg' => 'oops', 'details' => { 'class' => 'RuntimeError' })
-        expect(result['result']['_error']).not_to include('stack_trace')
       end
     end
   end
 
-  context 'when raising errors' do
-    let(:target) { conn_info('ssh') }
-    let(:echo_task) {
-      {
-        'name': 'echo',
-        'metadata': {
-          'description': 'Echo a message',
-          'parameters': { 'message': 'Default message' }
-        },
-        'file': {
-          'file_content': Base64.encode64("#!/usr/bin/env bash\necho $PT_message"),
-          'filename': "echo.sh"
-        }
-      }
-    }
+  describe 'action endpoint' do
+    # Helper to set the transport on a body hash, and then post
+    # to an action endpoint (/ssh/<action> or /winrm/<action>)
+    def post_over_transport(transport, action, body_defaults = {})
+      path = "/#{transport}/#{action}"
 
-    it 'returns non-html 404 when the endpoint is not found' do
-      body = {
-        'task': echo_task,
-        'target': {
-          'hostname': target[:host],
-          'user': target[:user],
-          'password': target[:password],
-          'port': target[:port],
-          'host-key-check': false
-        },
-        'parameters': { "message": "Hello!" }
-      }
+      target = conn_info(transport)
+      body = body_defaults.merge(target: {
+                                   hostname: target[:host],
+                                   user: target[:user],
+                                   password: target[:password],
+                                   port: target[:port]
+                                 })
 
-      post '/ssh/run_tasksss', JSON.generate(body), 'CONTENT_TYPE' => 'text/json'
-      expect(last_response).not_to be_ok
-      expect(last_response.status).to eq(404)
-      r = JSON.parse(last_response.body)
-      expect(r['msg']).to eq("Could not find route /ssh/run_tasksss")
-      expect(r['kind']).to eq("boltserver/not-found")
+      post(path, JSON.generate(body), 'CONTENT_TYPE' => 'text/json')
     end
 
-    it 'returns non-html 500 when the request times out' do
-      get '/500_error'
-      expect(last_response).not_to be_ok
-      expect(last_response.status).to eq(500)
-      r = JSON.parse(last_response.body)
-      expect(r['msg']).to eq('500: Unknown error: Unexpected error')
-      expect(r['kind']).to eq('boltserver/server-error')
+    describe 'run_task' do
+      it 'runs a simple echo task over SSH' do
```

Comment:
There are no available ssh targets on appveyor. You will need to filter this with something like `ssh: true` (similar to the :winrm, below). https://github.com/puppetlabs/bolt/blob/45056d869f7377e196e6b0aeb177d3515ed3d63f/Rakefile#L26-L30

Reply:
Gah, :woman_facepalming: Thank you @donoghuc!